circle.yml

```diff
@@ -41,7 +43,8 @@ test:
     - mkdir -p "$CIRCLE_TEST_REPORTS/$CIRCLE_PROJECT_REPONAME"
     - cd "$GO_PROJECT_SRC_PATH" && ./godelw verify --apply=false --tags none --junit-output="$CIRCLE_TEST_REPORTS/$CIRCLE_PROJECT_REPONAME/$CIRCLE_PROJECT_REPONAME-tests.xml"
     - mkdir -p "$CIRCLE_TEST_REPORTS/$CIRCLE_PROJECT_REPONAME-integration"
-    - cd "$GO_PROJECT_SRC_PATH" && ./godelw test --tags=integration --junit-output="$CIRCLE_TEST_REPORTS/$CIRCLE_PROJECT_REPONAME/$CIRCLE_PROJECT_REPONAME-integration.xml"
+    - cd "$GO_PROJECT_SRC_PATH" && ./godelw test --tags=integration --junit-output="$CIRCLE_TEST_REPORTS/$CIRCLE_PROJECT_REPONAME/$CIRCLE_PROJECT_REPONAME-integration.xml" -- -timeout 20m:
+      timeout: 10
```

Comment:
this will be moot after the tests stop running for forever, but i needed to override both circle's timeout and `go test`'s timeout. the circle timeout wasn't being respected by circle, so as a hack the stop tests print output after every subtest...